components/Voting/EloVote.styled.tsx

```diff
@@ -63,19 +68,30 @@ export const StyledSessionAbstract = styled('div')(({ theme }) => ({
 }))
 
 type StyledVoteButtonProps = {
-  kind: 'primary' | 'secondary'
+  kind: 'primary' | 'secondary' | 'tertiary'
```

Comment:
```suggestion
  kind: 'primary' | 'secondary' | 'tertiary'
  position: 'start' | 'middle' | 'end'
```

Reply:
could also consider something like `option: 'a' | 'draw' | 'b'`

Reply:
Yeah, it definitely depends on how reusable we want this component to be, currently a lot of the functionality isn't very explicit.

Using primary/secondary/tertiary also allows my hack for the button colours. I could use start/middle/end and then just create an object that maps the values as needed.